homeassistant/components/switch/rpi_rf.py

```diff
@@ -27,8 +27,10 @@
 DEFAULT_SIGNAL_REPETITIONS = 10
 
 SWITCH_SCHEMA = vol.Schema({
-    vol.Required(CONF_CODE_OFF): cv.positive_int,
-    vol.Required(CONF_CODE_ON): cv.positive_int,
+    vol.Required(CONF_CODE_OFF):
```

Comment:
You can do: vol.Required(CONF_CODE_OFF): [cv.positive_int] to use yaml list

If you prefer a comma-separated list use vol.All(cv.ensure_list_csv, [cv.positive_int])